checkScheduler.go

```diff
@@ -8,6 +8,9 @@ import (
 	"time"
 )
 
+// TODO Rename variables to clearly identifiable
+// Add, commit, push and create a pull request -> Assignee: alex kliimov
+
```

Comment:
Srsly?

Reply:
im a forgetful person i undo it until next push